nexus/test-utils/src/lib.rs

```diff
@@ -808,6 +808,8 @@ impl<'a, N: NexusServer> ControlPlaneTestContextBuilder<'a, N> {
                 cockroachdb_fingerprint: String::new(),
                 cockroachdb_setting_preserve_downgrade:
                     CockroachDbPreserveDowngrade::DoNotModify,
+                // TODO(ajs): Should we create this in RSS? it relies on policy
```

Comment:
I believe we have three options here:

1. Start the clickhouse clusters by enabling the policy with OMDB (not implemented yet) once the control plane boots. Reconfigurator will then bring the clusters up via manual OMDB blueprint generation. This requires no change to RSS, but will take a few rounds to bring up all nodes as shown the tests in this PR. We'll have to do this for deployed systems anyway, as they won't rerun RSS.
2. Add a policy knob to `rss_config.toml` and manually start all the zones with the right configuration and put them in the blueprint if the policy is enabled. It's actually unclear to me how this will be better, since Nexus still has to generate and push the configs down to `clickhouse-admin` inside the `ClickhouseServer` and `ClickhouseKeeper` zones.
3. Always enable the policy and do the zone config as in option 2.

I'm currently leaning towards 1 as it's the least amount of code and we have to do the manual work to bring up the nodes anyway via reconfigurator. If we wanted to get fancy, we may be able to figure out a way to generate the configuration for the clickhouse keeper and server nodes and trigger `clickhouse-admin` from RSS. That would at least make the startup automatic. But it would duplicate the work and complicate things a bit. Once we automate reconfigurator it will also be unnecessary.

Reply:
So, last week I actually changed the reconfigurator bits such that for new cluster configurations they can be brought up at once rather than keeper node by keeper node. This will reduce the iterations.